tests/edge-cases/all-01.txt

```diff
@@ -0,0 +1,2 @@
+brun '(all)'
+1
```

Comment:
Why not enforce that it has at least 1 parameter? I can't think of a circumstance where you would want to call it without any parameters.

Reply:
Yes, that's what I'm saying too. It's not useful with zero args.

Reply:
Here's one I can think of: when used with the new `apply`, you might do `(a #all ITEMS)`; it would save you checking against ITEMS being zero. Note that in python, `any` and `all` each accept zero parameters.

Reply:
Yes, makes sense if you are generating params with other code.